### ISSUE TITLE
[Bug]: Drawer moves with after right click

### ISSUE THREAD

**Issue by bitobeats** [corvudev/corvu#18]
### Bug description

When right clicking the Drawer component and then dismissing the system menu, the component acts as if I was dragging it.

### Reproduction Link

https://stackblitz.com/edit/corvu-mtkurb?file=src%2FApp.tsx

### Reproduction Steps

1. Open the drawer
2. Right click anywhere inside the drawer area
3. Left click inside the drawer area to dismiss the system menu that appears
4. Move the mouse

### Expected behavior

I expected that moving the mouse in step 5 wouldn't move the drawer around.

### Additional context

PS. Thank you for the lib! You are doing a great job for the community!!

**Comment by GiyoMoon** [comment]
Thanks a lot for reporting this bug! I'm currently busy with a small refactor of corvu (#17) and will fix this bug afterwards :)
Shouldn't take too long!

**Comment by bitobeats** [comment]
Thank you a lot! =)

On another topic: I realized the there's some css missing in the non-tailwind example of the Drawer. I'll try to find some time to send a PR correcting it during this week if I have some time, ok?

**Comment by GiyoMoon** [comment]
Sure! Thanks :)

**Comment by GiyoMoon** [comment]
Fixed in 1f759e3bf656393cdeb1260912278e65a75e0d80 and released as `@corvu/drawer@0.1.0`

**Comment by bitobeats** [comment]
Thank you for this! I just tested it and it seems to be working properly =].
Also, I loved the idea of splitting components in their own packages!

Just something I've noted: the "as" prop of the `<Drawer.Trigger>` used to accept components but now it only accepts html tags. 
By what I tested, it still works with components. Maybe it's just a type thing?

Though it would be better asking you if this was intentional before investigating
